frontend/src/components/EmployeeCards/EmployeeCards.tsx

```diff
@@ -156,27 +156,26 @@ const EmployeeCard = ({
 const EmployeeCards = () => {
   const { admins, drivers } = useEmployees();
 
-  const allEmployees = [...admins, ...drivers];
-  const adminIds = new Set(admins.map((admin) => admin.id));
-  const filteredEmployees = allEmployees.filter((employee: Employee) => {
-    // if not admin (means driver), check if another admin is representing this driver
-    if (employee['isDriver'] == undefined) return !adminIds.has(employee.id);
-    return true;
+  const employees: Record<string, DriverType | AdminType> = {};
+  [...admins, ...drivers].forEach((employee) => {
+    employees[employee.id] = { ...employees[employee.id], ...employee };
```

Comment:
I think one potential concern with this array destructuring is that fields other than `photolink` could be overridden as well if they differ between the driver and the admin. I'm not sure how often this would occur though.